packages/mcp/src/context-assembly/assembler.ts

```
@@ -0,0 +1,578 @@
+
+    if (this.config.scope.prioritizeRecent) {
+      filteredMemories.sort((a, b) => new Date(b.timestamp).getTime() - new Date(a.timestamp).getTime())
+    }
+
+    return filteredMemories
+  }
+
+  /**
+   * Create timeline summary from full timeline
+   */
+  private createTimelineSummary(timeline: any): TimelineSummary {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)